Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionBehaviour.cs

```diff
@@ -1504,18 +1504,18 @@ public enum GraspedMovementType {
 
         Joint[] joints = rigidbody.GetComponents<Joint>();
         foreach (var joint in joints) {
-          if (joint is FixedJoint) {
+          if (joint is FixedJoint && joint.connectedBody.isKinematic) {
```

Comment:
You need null checks for all of these since a joint doesn't _have_ to be connected to another body, it can be connected to the 'world'.